src/parser/ParsedQuery.h

```diff
@@ -271,6 +272,9 @@ class ParsedQuery {
   GraphPattern _rootGraphPattern;
   vector<SparqlFilter> _havingClauses;
   size_t _numGraphPatterns = 1;
+  // The number of additional internal variables that were added by the
+  // implementation of ORDER BY as BIND+ORDER BY.
+  int64_t numInternalVariables_ = 0;
```

Comment:
I think it currently does break because of a bug in the Subquery-Scoping (we just don't have a test case for this).

For now, add an interface that allows getting and setting this number, and use them when a subquery is set up.
Add comments `TODO<joka921> This is currently necessary because of the missing scoping of subqueries`